app/models/katello/repository.rb

```diff
@@ -739,6 +746,8 @@ def removable_unit_association
         self.ostree_branches
       elsif file?
         self.files
+      elsif def?
```

Comment:
I think you mean deb? :)

Reply:
Yes i guess so. I just applied your diff as it looked sane... ;)